docs/tr/week08/08-1.md

```diff
@@ -0,0 +1,204 @@
+
+<!-- Here we define the similarity metric between two feature maps/vectors as the cosine similarity. -->
+
+PIRL'in farkı evrişimsel öznitelik üreticisinin çıktısını doğrudan kullanmamasıdır. Aksine, evrişimsel öznitelik üreticisinden bağımsız katmanlar, $f$ ve $g$, kullanır. 
+
+
+<!-- What PIRL does differently is that it doesn't use the direct output of the convolutional feature extractor. It instead defines different _heads_ $f$ and $g$, which can be thought of as independent layers on top of the base convolutional feature extractor. -->
+
+Her şeyi bir araya getirirsek, PIRL'in NCE amaç fonksiyonu şu şekilde çalışır. Bir miniyığında(mini-batch), bir pozitif (benzer) çift ve çok sayıda negatif(benzer olmayan) çiftlere sahip olalım. Önce, dönüştürülmüş resmin öznitelik vektörü ile mini-batch'teki tüm örneklerin arasındaki benzerliği hesaplayalım. Sonra, pozitif çift için softmax-vari fonksiyonun skorunu hesaplayalım. Softmax skorunu maksimize etmemiz geri kalan tüm skorları minimize etmek demektir ki bu bizim enerji bazlı modellerde istediğimiz bir durumdur. Böylelikle, NCE kayıp fonksiyonu, benzer çiftlerin enerjilerini aşağı iterken benzer olmayan çiftlerin enerjilerini de yukarı itmemize olanak veren bir model tasarlamamızı sağlar.
+
+
+<!-- Putting everything together, PIRL's NCE objective function works as follows. In a mini-batch, we will have one positive (similar) pair and many negative (dissimilar) pairs. We then compute the similarity between the transformed image's feature vector ($I^t$) and the rest of the feature vectors in the minibatch (one positive, the rest negative). We then compute the score of a softmax-like function on the positive pair. Maximizing a softmax score means minimizing the rest of the scores, which is exactly what we want for an energy-based model. The final loss function, therefore, allows us to build a model that pushes the energy down on similar pairs while pushing it up on dissimilar pairs. -->
+
+Dr. LeCun, PIRL'ın çalışması için oldukça fazla sayıda negatif örneğe ihtiyaç uyulduğunu belirtmektedir. SGD'de mini-batchlerde negatif örneklerin çok sayıda parçasını sürekli olarak tutabilmek zor olabilir. Bü  durumu çözmek için PIRL, özel bir ön bellek yapısı kullanmaktadır.
+
+<!-- Dr. LeCun mentions that to make this work, it requires a large number of negative samples. In SGD, it can be difficult to consistently maintain a large number of these negative samples from mini-batches. Therefore, PIRL also uses a cached memory bank. -->
+
+**Soru**: L2 normu yerine neden kosinüs benzerliği kullanıyoruz?
+Answer:Çünkü, oldukça kısa(merkeze yakın) ya da oldukça uzun(merkezden uzak) iki vektör seçersek bu iki vektör L2 normuna göre birbirlerine benzer diyebiliriz. Bunun sebebi, L2 normunun vektörlerin elemanları arasındaki farklarının kareleri toplamıdır. Dolayısıyla, kosinüs benzerliği kullanarak sistemimizi "hile yapmamaya" yani vektörleri uzaltıp kısaltmadan iyi bir sonuç bulmaya zorluyoruz. 
```

Comment:
```suggestion
Answer:Çünkü, oldukça kısa (merkeze yakın) ya da oldukça uzun (merkezden uzak) iki vektör seçersek bu iki vektör L2 normuna göre birbirlerine benzer diyebiliriz. Bunun sebebi, L2 normunun vektörlerin elemanları arasındaki farklarının kareleri toplamasıdır. Dolayısıyla, kosinüs benzerliği kullanarak sistemimizi "hile yapmamaya" yani vektörleri uzaltıp kısaltmadan iyi bir sonuç bulmaya zorluyoruz. 
```

Reply:
toplamıdır is fine here.

Reply:
Resolving it, you can revert back to 'toplamıdır' for review PR.

Reply:
yeah my bad

Reply:
also we both blanked out on Answer: :)